### PR TITLE
Add signature to BrazeClient.identify()

### PR DESCRIPTION
**Pull request by devbrandonlong**

### CODE REVIEW

**Review by jywsn — APPROVED** [review]
Thank you!

File: pubspec.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: braze_plugin_web
 description: Unofficial Braze Plugin for Flutter for Web. It is mostly a Flutter Web Plugin acting as a wrapper around the official Braze Web SDK.
-repository: https://github.com/OncoHealth/braze_plugin_web
-version: 0.1.4
+repository: https:/jasper-health/braze_plugin_web_fork
+version: 0.1.6
```
Comment (devbrandonlong) [inline pubspec.yaml:4]: jumped to 0.1.6 because the app is already referencing `0.1.5` for some reason

File: pubspec.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: braze_plugin_web
 description: Unofficial Braze Plugin for Flutter for Web. It is mostly a Flutter Web Plugin acting as a wrapper around the official Braze Web SDK.
-repository: https://github.com/OncoHealth/braze_plugin_web
-version: 0.1.4
+repository: https:/jasper-health/braze_plugin_web_fork
```
Comment (jywsn) [inline pubspec.yaml]: Hmm.  I think this url is wrong.

`https://github.com/jasper-health/braze_plugin_web_fork`

Comment (devbrandonlong) [reply pubspec.yaml]: 🤦🏽‍♂️

**Comment by devbrandonlong** [comment]
0.1.5 was released in December, but the pubspec was never updated, so jumping to 0.1.6 in this MR

**Review by alysivji — APPROVED** [review]

**Pull request merged**